src/components/shared/IconWrapper.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const IconWrapper = ({children}) => {
+  return (
+    <span className='flex center icon'>
+        {children}
+    </span>
+  )
+}
```

Comment:
Liked how you made small components for things which you are going to reuse alot in future.